etc/tasks/deleteEtherpadSessions.js

```diff
@@ -0,0 +1,78 @@
+/*!
```

Comment:
Is this script something that we'll need to run manually from time to time? How difficult would it be to make sure that the sessions are cleaned up properly?